store/src/main/resources/application.conf

```diff
@@ -19,19 +19,19 @@ akka.persistence {
       tables-autocreate = true
       keyspace-autocreate = true
       replication-factor = 1
-      keyspace = "tenant_service"
+      keyspace = "store_service"
     }
 
     snapshot {
       tables-autocreate = true
       keyspace-autocreate = true
       replication-factor = 1
-      keyspace = "tenant_service"
+      keyspace = "store_service"
     }
   }
 }
 datastax-java-driver.advanced.reconnect-on-init = true
 datastax-java-driver {
-  basic.contact-points = ["scylla-db:9042"]
+  basic.contact-points = ["10.152.183.216:9042"]
```

Comment:
yikes, a hard-coded IP

Reply:
Yeah, I had to do this to get the success I posted, but it wasn't supposed to be pushed. Will revert